### PR TITLE
Add boarding passes to Leg

### PR DESCRIPTION
**Pull request by kantord**

### CODE REVIEW

File: src/flight/types/outputs/Leg.js [hunk 1]
```diff
@@ -106,5 +108,23 @@ export default new GraphQLObjectType({
...
+        { id, bookingId }: { id: string, bookingId: string },
+        _,
+        { dataLoader }: GraphqlContextType,
+      ) => {
+        const booking = await dataLoader.booking.load(bookingId);
+        const boardingPasses = idx(booking, _ => _.assets.boardingPasses);
+
+        return {
+          flightNumber: id,
+          boardingPassUrl: boardingPasses && boardingPasses[id],
```
Comment (tbergq) [inline src/flight/types/outputs/Leg.js]: how about moving this ti idx
`const boardingPassUrl = idx(booking, _ => _.assets.boardingPasses[id]);`

Comment (kantord) [reply src/flight/types/outputs/Leg.js]: :man_shrugging: dunno. The your idx version looks fine to me, however I don't see the added value if there's no further nesting. We can add it if you want :man_shrugging:

Comment (tbergq) [reply src/flight/types/outputs/Leg.js]: The added value would be that you don't have to do `boardingPasses && boardingPasses[id]` after using idx,  you can just do this really

```
return {
        flightNumber: id,
        boardingPassUrl: idx(booking, _ => _.assets.boardingPasses[id]);
}
```

**Comment by tbergq** [comment]
Also flight query crashes if you add boarding pass to legs

**Comment by kantord** [comment]
Crash fixed an tests added

**Pull request merged**